app/controllers/unaccompanied_controller.rb

```diff
@@ -209,16 +209,16 @@ def handle_step
     if params["stage"].to_i == SPONSOR_DATE_OF_BIRTH
       # There must be a better way!
       begin
-        minor_dob = Date.new(params["unaccompanied_minor"]["sponsor_date_of_birth_year"].to_i, params["unaccompanied_minor"]["sponsor_date_of_birth_month"].to_i, params["unaccompanied_minor"]["sponsor_date_of_birth_day"].to_i)
+        sponsor_dob = Date.new(params["unaccompanied_minor"]["sponsor_date_of_birth_year"].to_i, params["unaccompanied_minor"]["sponsor_date_of_birth_month"].to_i, params["unaccompanied_minor"]["sponsor_date_of_birth_day"].to_i)
 
-        if minor_dob > 18.years.ago.to_date
-          @application.errors.add(:sponsor_date_of_birth_day, I18n.t(:too_young_date_of_birth, scope: :error))
+        if sponsor_dob > 18.years.ago.to_date
```

Comment:
```suggestion
        if 18.years.ago.to_date < sponsor_dob
```
The original way gave me a wtf moment - WDYT?